soc_026_rw0_global_gender_gap/README.md

```diff
@@ -0,0 +1,20 @@
+## Gender Gap Index Dataset Pre-processing
+This file describes the data pre-processing that was done to the [Gender Gap Index](http://www3.weforum.org/docs/WEF_GGGR_2021.pdf) for [display on Resource Watch](https://resourcewatch.org/data/explore/).
+
+The source provides the data in a pdf that is reformated into a table. Below, we describe the steps used to download the Global Gender Gap index and four subindices from the pdf and upload the reformated table to Carto.
+
+1. Read in the necessary tables from the WEF pdf using the tabula-py module.
+2. Create a dictionary: keys are each year of data downloaded, values are the dataframes corresponding to each year.
+3. The first dataframe pulled using Tabula is the main Global Gender Gap Index, but is split into two sets of columns. The two sets of columns are then concatenated for each year, and a new column is created for year.
+4. The next two dataframes contain the four subindicies. Each dataframe contains two indicies, with each index split into two sets of columns. The two sets of columns are then concatenated for each year, and a new column is created for year.
+5. For each year of data pulled, the main index and subindicies are merged by ['year', 'country'].
+6. This dataframe is then appended to the Carto table 
```

Comment:
We don't need to go into much detail in these steps about splitting into columns, etc. Just a big picture of how you changed the data from what you might find at the source.
```suggestion
1. Read in the data tables for the global gender gap index and its four subindices from the Global Gender Gap Report pdf using the tabula-py module. Both the 2020 and 2021 datasets were pulled in.
2. Transform the data for each index/subindex into a pandas dataframe.
3. A year column was added to all dataframes to indicate the year of the report.
4. Country names were processed to fix inconsistencies in naming.
5. Dataframes containing the main index and subindicies were merged into one table using the columns 'year' and 'country'.
6. This dataframe was then appended to the existing data we had previously processed by reading in the existing Carto table as a dataframe and concatenating it with the new data.
```